spec/spec_helper.rb

```diff
@@ -119,27 +119,19 @@ def fixture_file_accessor(spec_or_name, platform = Pod::Platform.ios)
 def fixture_target_definition(name = 'Pods', platform = Pod::Platform.ios)
   platform_hash = { platform.symbolic_name => platform.deployment_target }
   parent = Pod::Podfile.new
-  Pod::Podfile::TargetDefinition.new(name, parent,
-                                     'abstract' => false,
-                                     'name' => name,
-                                     'platform' => platform_hash)
+  Pod::Podfile::TargetDefinition.new(name, parent, 'abstract' => false, 'name' => name, 'platform' => platform_hash)
```

Comment:
line is actually shorter than other lines in here, let me know if you want to have multiple lines